tests/functional/spec/components/checkbox/mutually_exclusive/mutually_exclusive_dropdown.spec.js

```diff
@@ -45,6 +45,7 @@ describe('Component: Mutually Exclusive Dropdown With Single Checkbox Override',
 
         // When
         .selectByValue(DropdownPage.dropdown(), 'Liverpool')
+        .click(DropdownPage.dropdownExclusiveIPreferNotToSayLabel())
```

Comment:
By clicking the label you may be effectively clicking the checkbox so this might not be the correct fix